VERSION

```diff
@@ -1 +1 @@
-0.0.1
\ No newline at end of file
+0.0.2
```

Comment:
I might be missing something, but this doesn't appear to be in-line with https://github.com/sparkgeo/stac-validator/releases

Reply:
Ahh. Good catch